src/js/controllers/import.js

```diff
@@ -25,15 +27,15 @@ angular.module('copayApp.controllers').controller('importController',
 
       if ($scope.isChromeApp) {
```

Comment:
Should compare as https://github.com/bitpay/copay/pull/5797/files#diff-c5f2d6282049bf9d04bcb177c5912451R64